vega/utils.py

```diff
@@ -2,18 +2,24 @@
 import codecs
 import collections
 import os.path
+import copy
 
 import pandas as pd
 
 
-def update(d, u):
-    """Update dictionary."""
+def update(d, u, overwrite=True):
```

Comment:
Instead of adding this option, why don't we just reorder the arguments in the call?